👨‍🌾 Fails to build when FastRTPS is enabled on Rolling

Builds are failing on the Rolling nightly performance job (https://build.ros2.org/job/Rci__nightly-performance_ubuntu_focal_amd64/280/) with the error message
```
In file included from /tmp/ws2/src/performance_test/performance_test/src/communication_abstractions/resource_manager.cpp:15:
/tmp/ws2/src/performance_test/performance_test/src/communication_abstractions/resource_manager.hpp:19:12: fatal error: fastrtps/participant/Participant.h: No such file or directory
   19 |   #include <fastrtps/participant/Participant.h>
```

Locally I can see the build flags have changed from 
```
-I/tmp/ws/install_isolated/fastrtps/include -I/tmp/ws/install_isolated/fastrtps/include/fastrtps/include
```
to
```
-I/fastrtps/include
```

I think it's related to https://github.com/ros2/rosidl_typesupport_fastrtps/pull/68.

---
https://github.com/osrf/buildfarmer/issues/192

--- a/performance_test/CMakeLists.txt
+++ b/performance_test/CMakeLists.txt
@@ -64,9 +64,7 @@ endif()
 ament_export_include_directories(include)
 ament_export_dependencies(rosidl_default_runtime)
 
-# This is a workaround for broken include paths on some systems.
-include_directories(${FastRTPS_INCLUDE_DIR} ${FastRTPS_INCLUDE_DIR}/fastrtps/include ${fastcdr_INCLUDE_DIR})
-include_directories(include ${osrf_testing_tools_cpp_INCLUDE_DIR})
+include_directories(include)
 
 # FastRTPS
 option(PERFORMANCE_TEST_FASTRTPS_ENABLED "Enable FastRTPS" OFF)
@@ -316,7 +314,7 @@ endif()
 
 if(PERFORMANCE_TEST_FASTRTPS_ENABLED)
   target_link_libraries(${EXE_NAME}
-      ${FastRTPS_LIBRARIES}
+      fastrtps
       fast_rtps_idl
       fastcdr)
 endif()

--- a/performance_test/package.xml
+++ b/performance_test/package.xml
@@ -13,6 +13,7 @@
 
   <build_depend>builtin_interfaces</build_depend>
   <build_depend>boost</build_depend>
+  <build_depend>fastrtps</build_depend>
   <build_depend>fastrtps_cmake_module</build_depend>
   <build_depend>java</build_depend>
   <build_depend>maven</build_depend>

